_specification-v1/imageset.md

```diff
@@ -0,0 +1,109 @@
+---
+title: imageset.json Manifest Specification
+description: A specification which describes the required structure of an imageset.json manifest file.
+cta: Read the imageset manifest spec
+
+# Navigation config
+nav_display: true
+nav_label: Imageset
+nav_heading_selector: h1, h2, h3
+---
+
+# {{page.title}}
+
+`imageset.json` is a <a href="https://www.json.org/" class="o-typography-link--external"><abbr title="JavaScript Object Notation">JSON</abbr></a> format file that is responsible for describing various aspects of an Origami Imageset.
+
+## Properties
+
+### sourceDirectory
+
+<table class="o-manifest__table o-table o-table--compact o-table--row-headings o-table--vertical-lines o-table--horizontal-lines" data-o-component="o-table">
+	<tr>
+		<th scope="row" role="rowheader">Type</th>
+		<td><code>String</code></td>
+	</tr>
+	<tr>
+		<th scope="row" role="rowheader">Required</th>
+		<td><code>true</code></td>
+	</tr>
+</table>
+
+**Must** be a the relative path to the folder which contains the images within the imageset.
+<pre><code class="o-syntax-highlight--json">{
+	"sourceDirectory": "svg"
+}</code></pre>
+
+### scheme
+
+<table class="o-manifest__table o-table o-table--compact o-table--row-headings o-table--vertical-lines o-table--horizontal-lines" data-o-component="o-table">
+	<tr>
+		<th scope="row" role="rowheader">Type</th>
+		<td><code>String</code></td>
+	</tr>
+	<tr>
+		<th scope="row" role="rowheader">Required</th>
+		<td><code>true</code></td>
+	</tr>
+</table>
+
+Defines the custom url scheme for images in this imageset when requested via the Origami Image Service, this **must**** be a unique name across all the imagesets:
+<pre><code class="o-syntax-highlight--json">{
+	"scheme": "ftanimal"
+}</code></pre>
+
+### images
+
+<table class="o-manifest__table o-table o-table--compact o-table--row-headings o-table--vertical-lines o-table--horizontal-lines" data-o-component="o-table">
+	<tr>
+		<th scope="row" role="rowheader">Type</th>
+		<td><code>Array</code></td>
+	</tr>
+	<tr>
+		<th scope="row" role="rowheader">Required</th>
+		<td><code>true</code></td>
+	</tr>
+</table>
+
+A list of image objects for each image in the imageset.
+Each object in the list accepts the following properties:
+
+**required**:
+- `name`: type `String`. The name which will be used together with the <a href="#scheme">scheme</a> to requested the image via the Origami Image Service
+- `extension`: type `String`. The extension for the file type of the image
+- `path`: type `String`. The relative path to the image within the imageset
+- `previousHash`: type `String` or `null` or `undefined`. The previous hash of the image, this will be `null` or `undefined` if the image has never been updated
+- `hash`: type `String`. The hash of the image
```

Comment:
Should we define the type of hash? And maybe put an aside to [origami-image-set-tools](https://github.com/Financial-Times/origami-image-set-tools)?

Reply:
it's a sha-512 hash, good point to mention that this whole section of the manifest is created automatically by oist